joint_limits_interface/CHANGELOG.rst

```diff
@@ -0,0 +1,247 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package joint_limits_interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
IMO you could drop the ROS1 changelog, especially since the versions won't line up

Reply:
Makes sense. done.